x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table.tsx

```diff
@@ -0,0 +1,155 @@
+  error?: string;
+  loading: boolean;
+}
+
+export const SpikeAnalysisTable: FC<Props> = ({ changePointData, error, loading }) => {
+  const [pageIndex, setPageIndex] = useState(0);
+  const [pageSize, setPageSize] = useState(10);
+
+  const columns: Array<EuiBasicTableColumn<ChangePoint>> = [
+    {
+      field: 'score',
+      name: (
+        <>
+          {i18n.translate(
+            'xpack.aiops.correlations.failedTransactions.correlationsTable.pValueLabel',
+            {
+              defaultMessage: 'Score',
+            }
+          )}
+        </>
+      ),
+      align: RIGHT_ALIGNMENT,
+      render: (_, { score }) => {
+        return (
+          <>
+            <ImpactBar size="m" value={Number(score.toFixed(2))} label={score.toFixed(2)} />
+          </>
+        );
+      },
+      sortable: true,
+    },
+    {
+      field: 'pValue',
+      name: (
+        <>
+          {i18n.translate(
+            'xpack.aiops.correlations.failedTransactions.correlationsTable.impactLabel',
+            {
+              defaultMessage: 'Impact',
+            }
+          )}
+        </>
+      ),
+      render: (_, { pValue }) => {
+        const label = getFailedTransactionsCorrelationImpactLabel(pValue);
+        return label ? <EuiBadge color={label.color}>{label.impact}</EuiBadge> : null;
+      },
+      sortable: true,
+    },
+    {
+      field: 'fieldName',
+      name: i18n.translate(
+        'xpack.aiops.correlations.failedTransactions.correlationsTable.fieldNameLabel',
+        { defaultMessage: 'Field name' }
+      ),
+      sortable: true,
+    },
+    {
+      field: 'fieldValue',
+      name: i18n.translate(
+        'xpack.aiops.correlations.failedTransactions.correlationsTable.fieldValueLabel',
+        { defaultMessage: 'Field value' }
+      ),
+      render: (_, { fieldValue }) => String(fieldValue).slice(0, 50),
+      sortable: true,
+    },
+    {
+      field: 'pValue',
+      name: 'p-value',
+      render: (pValue: number) => pValue.toPrecision(3),
+      sortable: true,
+    },
+  ];
+
+  const onChange = useCallback((tableSettings) => {
+    const { index, size } = tableSettings.page;
+
+    setPageIndex(index);
+    setPageSize(size);
+
+    // onTableChange(tableSettings);
+  }, []);
+
+  const { pagination, pageOfItems } = useMemo(() => {
+    const pageStart = pageIndex * pageSize;
+
+    const itemCount = changePointData?.length ?? 0;
+    return {
+      pageOfItems: changePointData?.slice(pageStart, pageStart + pageSize),
+      pagination: {
+        pageIndex,
+        pageSize,
+        totalItemCount: itemCount,
+        pageSizeOptions: PAGINATION_SIZE_OPTIONS,
+      },
+    };
+  }, [pageIndex, pageSize, changePointData]);
+
+  return (
+    <EuiBasicTable
+      columns={columns}
+      items={pageOfItems ?? []}
+      noItemsMessage={noDataText}
+      onChange={onChange}
+      pagination={pagination}
+      loading={loading}
+      error={error}
+      // sorting={sorting}
```

Comment:
Have you left this in for a follow-up to add in row selection functionality?

Reply:
Yes 👍